openspec/changes/archive/2026-04-23-rich-output-rendering/design.md

```
@@ -0,0 +1,265 @@
+# Rich Output Rendering — Design
+
+## Architecture
+
+Two independent subsystems, connected by convention (the ` ```svg ` fenced
+block language tag):
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)